### PR TITLE
Roy/update readme

### PR DESCRIPTION
**Pull request by royanger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+## Getting Started
+
+Clone the repository and install the dependencies.
+
+```bash
+git clone github.com/clerk/migration-script
+
+cd migration-script
+
+bun install
```
Comment (JacobMGEvans) [inline README.md:16]: Let's not, I rather it be more Node-oriented, Bun is a dependency so people don't need to install new tooling just whatever Node package manager they have. We can also gitignore the lockfile (normal practice in OSS and certain tooling)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+```bash
+CLERK_SECRET_KEY=your-secret-key
+```
+
+### Run the script
+
+```bash
+bun ./index.ts
+```
```
Comment (JacobMGEvans) [inline README.md:71]: use the package.json script

**Review by JacobMGEvans — CHANGES_REQUESTED** [review]
Looking good, first pass

File: README.md (hunk 1 above)
Comment (royanger) [reply README.md:16]: You want to ignore all lock files or just the bun one?

Comment (JacobMGEvans) [reply README.md]: ignore all of the `*-lockfile` or whatever

**Pull request merged**